.github/workflows/ms.analysisservices.servers.yml

```
@@ -36,23 +36,29 @@ env:
 
 jobs:
   ############################
-  #   SET INPUT PARAMETERS   #
+  #   PREPARE PIPELINE   #
   ############################
-  job_set_workflow_param:
+  job_prepare_pipeline:
     runs-on: ubuntu-20.04
-    name: 'Set input parameters to output variables'
+    name: 'Prepare pipeline'
```

Prepare or initialize?
```suggestion
    name: 'Initialize pipeline'
```

---

Isn't it more of a preparation as we fetch input parameters & search for parameter files - to be able to run the pipeline in the first place?